backend/src/main/java/com/exadel/carinsurance/utils/NoOpPasswordEncoder.java

```diff
@@ -0,0 +1,22 @@
+package com.exadel.carinsurance.utils;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+public class NoOpPasswordEncoder implements PasswordEncoder {

```

Comment:
NoOp means `Non Operational` and used for implementations that do nothing. In your case you actually have a logic and actual validations in some of the cases. So please consider renaming it to something more appropriate as `TokenFilteringPasswordEncoder `

Reply:
Done